CONTRIBUTING.md

```diff
@@ -1,11 +1,49 @@
+- Face-to-face topic-based work meetings will be announced at least 4 weeks in advance. Ideally, they are announce 8 weeks in advance as defined in the W3C process model. 
+- Proposed changes to any document MUST be handled via a Pull Request (PR). If the change came up during a meeting the meeting owner is responsible for creating the PR.
+- PRs will be reviewed during official meetings, but off-line reviews (LGTMs, NOT LGTMs) and comments are strongly encouraged to gauge the group's opinion on the proposed change prior to the meeting.
+- If a topic is not defined specifically enough yet to justify a PR against the specification, contributors are encouraged to file an issue. 
+- During meetings, priority will be given to PRs that appear to be ready for a vote over those that appear to require discussions.
+- Rebases, typo fixes, etc. do not count as substantial. Note, administrivia PRs that do not materially modify output documents may be processed by admins as needed.
+- PR's can be marked for resolution during a meeting and will be merged when all issues have been addressed and the PR was open for comments for at least one business week. 
+- If issues arise after a PR has been merged, the preferred approach is to file a new issue. In rare cases the group can decide to revert a PR.
+
+## Participation  
+
+There are three categories of project participation:
+
+1. Non Working Group Member. This is anyone who  participates in the group's activities in any of our communication channels (email, github issues/PRs, meetings, etc.). In contrast to Working Group members they can file a PR but not vote on a PR. They can, however, create issues for specifications. 
+2. Working Group Member. This is anyone who officially participates in the group's activities in any of our communication channels (email, github issues/PRs, meetings, etc.). Working group members can file PRs against documents and can participate in the PR voting procedure. Working Group members need to be employed by a W3C member organization or participate as invited experts. The [W3C process document](https://www.w3.org/2018/Process-20180201/#MemberBenefits) defines the rights of members. 
+3. Chairs and Co-Chairs are members of the group but have the ability to perform administrative actions on behalf of the group. For example, manage the website, github repos and moderate the meetings. Their actions should be done with the knowledge and consent of the group. They also have the ability to merge/close PRs, but only per the group's approval. 
+
+## PRs and Voting
+
+Specificaiton changes are proposed via PRs which are expected to meet the following criteria prior to being merged:
+
+- The author of the PR indicates that it is ready for review by asking for it to be discussed in an upcoming meeting - eg. by adding it to the agenda document.
```

Comment:
Perhaps there might be another form of indication like remove `[WIP]` from the title?